packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -444,7 +442,7 @@ class AnimationController extends Animation<double>
   /// During the animation, [status] is reported as [AnimationStatus.forward],
   /// which switches to [AnimationStatus.completed] when [upperBound] is
   /// reached at the end of the animation.
-  TickerFuture forward({ double? from }) {
+  TickerFuture forward({double? from}) {
```

Comment:
we're not consistent with this one, also I'm a bit indifferent

Reply:
I think @a14n has done things to make this more consistent but it's mostly manually handled.

Reply:
Yeah it was done (automatically with a combination of analyzer and scripts) at some point in flutter history.